spec/features/availability_spec.rb

```diff
@@ -146,4 +146,24 @@
       end
     end
   end
+
+  context 'when using Alma' do
+    before do
+      allow(Rails.configuration).to receive(:use_alma).and_return(true)
+    end
+
+    describe 'Electronic Holdings are displayed on a record page', js: true do
```

Comment:
should these spec titles be like "don't display"? This one and line 162?

Reply:
Good catch. Fixed.